api/generator/generator.go

```diff
@@ -150,6 +151,7 @@ func (g *Handler) Handle(w http.ResponseWriter, r *http.Request) error {
 // sending the CSR to the server.
 type CertGeneratorHandler struct {
 	generator *csr.Generator
+	bundler   *bundler.Bundler
```

Comment:
The point is to avoid always having two API calls, each one possibly routed to different backend CAs.